pkg/kubelet/allocation/features_linux.go

```diff
@@ -35,3 +35,7 @@ func IsInPlacePodVerticalScalingAllowed(pod *v1.Pod) (allowed bool, msg, reason
 	}
 	return true, "", ""
 }
+
+func IsInPlacePodLevelResourcesVerticalScalingAllowed(pod *v1.Pod) (allowed bool) {
```

Comment:
I'm a little surprised not to see any changes in `handlePodResourcesResize`. Does this need to be called from there as part of the feasibility checks?